packages/okta-adapter/src/adapter_creator.ts

```diff
@@ -43,6 +45,27 @@ const credentialsFromConfig = (config: Readonly<InstanceElement>): Credentials =
   }
 }
 
+const validateOktaFetchConfig = ({
+  fetchConfig,
+  clientConfig,
+  apiDefinitions,
+  privateApiDefinitions,
+}:
+{ fetchConfig: OktaFetchConfig
```

Comment:
indentation nit (start and finish should be aligned):
```suggestion
}: { fetchConfig: OktaFetchConfig
```